code/modules/projectiles/projectile/special.dm

```diff
@@ -227,22 +227,24 @@
 	var/datum/gas_mixture/environment = proj_turf.return_air()
 	if(environment)
 		var/pressure = environment.return_pressure()
-		if(pressure < 30)
+		if(pressure < 60)
 			name = "full strength plasma blast"
-			damage *= 3
+			damage *= 4
 	..()
 
 /obj/item/projectile/plasma/on_hit(atom/target)
 	. = ..()
 	if(istype(target, /turf/closed/mineral))
 		var/turf/closed/mineral/M = target
 		M.gets_drilled(firer)
-		range = max(range - 1, 1)
-		return -1
+		Range()
+		if(range > 0)
+			return -1
```

Comment:
I know, but why did you change the code to that? Afaik Range() is already called a bit after on_hit() in projectile code. Is your change fixing a specific bug?